src/rulesMatcher.es6

```diff
@@ -1,10 +1,15 @@
+/**
+ * Demo - https://regex101.com/r/AA4xaq/2
+ */
+const suitRegex = /^\.(?:[a-z0-9]*-)?[A-Z](?:[a-zA-Z0-9]+)\b(?!\[|:|\.is-|--)(?:-[a-zA-Z0-9]+)?/;
```

Comment:
`(?!\[|:|\.is-|--)` is not necessary anymore if you add `$` at the and of the regexp